### PR TITLE
Readme changes

### PR DESCRIPTION
**Pull request by vividos**
### Description of Change ###

Fixed some docs links to use en-us and adjusted Xamarin.iOS supported versions.

### Issues Resolved ### 

None.

### Platforms Affected ### 

None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ originated from this plugin. Here's a little guide on how to do it.
 1. If you don't have the Xamarin.Essentials NuGet package installed yet, install
    it into the Android, iOS and UWP projects. Also install it into your Forms
    project, if you're calling the FilePicker from there. Be sure to also
-   properly [initialize Essentials](https://docs.microsoft.com/de-de/xamarin/essentials/get-started).
+   properly [initialize Essentials](https://docs.microsoft.com/en-us/xamarin/essentials/get-started).
```
Comment (jfversluis) [inline README.md]:
```suggestion
   properly [initialize Essentials](https://docs.microsoft.com/xamarin/essentials/get-started).
```

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Simple cross-platform plug-in that allows you to pick files and work with them.
 
 The original project can be found [here](https://github.com/Studyxnet/FilePicker-Plugin-for-Xamarin-and-Windows/), but seems abandoned, this one was forked and further developed.
 
-### The future: [Xamarin.Essentials](https://docs.microsoft.com/de-de/xamarin/essentials/)
+### The future: [Xamarin.Essentials](https://docs.microsoft.com/en-us/xamarin/essentials/)
```
Comment (jfversluis) [inline README.md]:
```suggestion
### The future: [Xamarin.Essentials](https://docs.microsoft.com/xamarin/essentials/)
```

Comment (jfversluis) [reply README.md]: If you leave it out entirely, it will automatically select the right one for the user :)

**Review by jfversluis — DISMISSED** [review]

**Comment by vividos** [comment]
Thanks for merging this!

**Pull request merged**